frontend/src/app/game/components/graphs/line-graph/line-graph.component.ts

```diff
@@ -70,13 +70,24 @@ export class LineGraphComponent implements OnInit, AfterViewInit {
   private seriesLength = 0;
   private lastValue: number | undefined;
 
+  private axesFontSize = 11;
+  private readonly widthThresholds = {
+    iPhone8: 500,
```

Comment:
Is it only for Apple devices? Should not be.

Reply:
It does not matter what is the name of this variable. I named it just as typed resolution